prelude.links

```diff
@@ -502,6 +502,22 @@ fun asList(t) server {
   for (x <-- t) [x]
 }
 
+# Gets children of elements in an XML forest, concatenated in order
+# @param {Xml} xml
+# @returns {Xml}
+sig childNodes : (Xml) ~> Xml
```

Comment:
Changelog: generalised behaviour

Reply:
(should we adopt the habit of extending the changelog as we go rather than all at once just before a release?)

Reply:
Not sure, but for 0.9.4 it seems that there will be plenty of PRs that haven't been doing this so it might be confusing to start halfway through.  Shall we discuss at the next meeting?